README.md

```diff
@@ -5,102 +5,165 @@
 It runs with [Prow CI](https://github.com/kubernetes/test-infra/blob/master/prow/) on a [Google Kubernetes Engine Cluster](https://cloud.google.com/kubernetes-engine/).
 It is designed to support adding more k8s providers.
 
+## How to run tests manually
+
+### Prerequisites
+---
+
+#### Create a k8s cluster
+
+- Create a new project on Google Cloud.
+
+- Create a Service Account on GKE with role `Kubernetes Engine Service Agent` & `Kubernetes Engine Admin` and download the json file.
+
+- Set the following environment variables & deploy cluster.
+```
+export PROJECT_ID=<google-cloud project-id>
+export CLUSTER_NAME=prombench
+export ZONE=us-east1-b
+export AUTH_FILE=<path to service-account.json>
+
+./prombench gke cluster create -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID \
+    -v ZONE:$ZONE -v CLUSTER_NAME:$CLUSTER_NAME -f components/prow/cluster.yaml
+```
+
+### Deploy Prometheus-Meta & Grafana
+---
+
+This is used for collecting and displaying the test results.
+
+- Set the following environment variables
+```
+export GCLOUD_SERVICEACCOUNT_CLIENTID=<email-id present in service-account.json>
+export GRAFANA_ADMIN_PASSWORD=password
+```
+**Note:** The `GCLOUD_SERVICEACCOUNT_CLIENTID` is used to grant cluster-admin-rights to the service-account. This is needed to create RBAC roles on GKE.
+
+- Deploy the [nginx-ingress-controller](https://github.com/kubernetes/ingress-nginx) which will be used to access Prometheus-Meta & Grafana.
+```
+./prombench gke resource apply -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID -v ZONE:$ZONE \
+    -v CLUSTER_NAME:$CLUSTER_NAME -v GCLOUD_SERVICEACCOUNT_CLIENTID:$GCLOUD_SERVICEACCOUNT_CLIENTID \
+    -f components/prow/manifests/rbac.yaml -f components/prow/manifests/nginx-controller.yaml
+```
+
+- Export the nginx-ingress-controller IP address.
+```
+kubectl get ingress ing -o go-template='{{ range .status.loadBalancer.ingress}}{{.ip}}{{ end }}'
```

Comment:
this will not work without 
// Generate auth config so we can use kubectl.
gcloud container clusters get-credentials $CLUSTER_NAME --zone=$ZONE